en/cpp/server_plugins.md

```diff
@@ -1,11 +1,23 @@
 # Server Plugins
```

Comment:
@bazfp @julianoes My feeling is that we were talking too much about the difference between server and client and obscuring the point:  "these let you add a mavlink service to your non mavlink flight stack". 

If you hate this we can revert the commit.